blocks/contact/contact.js

```diff
@@ -0,0 +1,107 @@
+                                                    </li>
+                                                    <li class="profile__contact-item">
+                                                        <a aria-label="E-Mail"
+                                                            class="plain-link profile__contact-item-link"
+                                                            href="mailto:${item.Email}" title="E-Mail">
+                                                            <span class="icon icon-mail icon--symbol">
+                                                            </span>
+                                                            <span
+                                                                class="profile__contact-item-label">${item.Email}</span>
+                                                        </a>
+                                                    </li>
+                                                    <li class="profile__contact-item">
+                                                        <a aria-label="vCard"
+                                                            class="plain-link profile__contact-item-link"
+                                                            href="${item.vCard}"
+                                                            target="_blank" title="vCard">
+                                                            <span class="icon icon-file-download">
+                                                            </span>
+                                                            <span class="profile__contact-item-label">vCard
+                                                                herunterladen</span>
+                                                        </a>
+                                                    </li>
+                                                </ul>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>`).join('')}
+                    </div>
+                    <div class="grid__structure profile-collection__structure profile-collection__structure--secondary">
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>`;
+}
+
+export default async function decorate(block) {
+  const contacts = [];
+  [...block.firstElementChild.firstElementChild.children].forEach((p) => {
+    const child = p.querySelector('a');
+    contacts.push(child.href);
+  });
+
+  const reqs = await Promise.all(contacts.map((contact) => fetch(contact)));
+
+  const items = [];
+  // eslint-disable-next-line no-restricted-syntax
+  for (const req of reqs) {
+    if (req.ok) {
+      // eslint-disable-next-line no-await-in-loop
+      const res = await req.text();
+      const parser = new DOMParser();
```

Comment:
You can also do 
```
const doc = document.createRange().createContextualFragment(res);
```
so you don't generate a full html document. 

`createContextualFragment('<h1>Hello</h1>')`
generates a fragment with
```
<h1>Hello</h1>
```
while `parser.parseFromString('<h1>Hello</h1>', 'text/html');`

generates
```
<html>
  <head></head>
  <body>
    <h1>Hello</h1>
  </body>    
</html>
```